test/integration/userclient_test.go

```diff
@@ -411,6 +411,9 @@ func TestUserInitialization(t *testing.T) {
 			t.Fatalf("Could not clean up users: %v", err)
 		}
 		if _, err := etcdClient.Delete(context.Background(), path.Join(masterConfig.EtcdStorageConfig.OpenShiftStoragePrefix, "/useridentities"), &etcdclient.DeleteOptions{Recursive: true}); err != nil && !etcdutil.IsEtcdNotFound(err) {
+			t.Fatalf("Could not clean up useridentities: %v", err)
+		}
+		if _, err := etcdClient.Delete(context.Background(), path.Join(masterConfig.EtcdStorageConfig.OpenShiftStoragePrefix, "/identities"), &etcdclient.DeleteOptions{Recursive: true}); err != nil && !etcdutil.IsEtcdNotFound(err) {
```

Comment:
@liggitt @deads2k this is scary change, has something changed in etcd?

With this change this integration test pass, without it I see a lot of:

```
	userclient_test.go:428: claim with existing mapped identity with invalid user UID: Could not create identity: identities.user.openshift.io "idp:bob" already exists
	userclient_test.go:465: add missing identity and user with preferred username and display name: Expected success, got error 'users.user.openshift.io "bob" not found'
	userclient_test.go:465: add missing identity and user with preferred username and display name: Expected success, got error 'users.user.openshift.io "bob" not found'
	userclient_test.go:465: add missing identity and user with preferred username and display name: Expected success, got error 'users.user.openshift.io "bob" not found'
...
...
```